Documentation/UsingSetting/UserTSconfig.rst

```diff
@@ -200,27 +200,26 @@ When a page TSconfig property is set in **user** TSconfig that way, regardless
 of whether it is in the TSconfig field of a
 group or a user, it **overrides** the value of the according **page** TSconfig property.
 
-To illustrate this feature let's say the action
-:guilabel:`Web > Info > Localization Overview` has been disabled via Page
-TSconfig:
+To illustrate this feature let's say new pages and copied pages are not hidden
+by default:
 
 ..  code-block:: typoscript
     :caption: EXT:site_package/Configuration/page.tsconfig
 
-    mod.web_info.menu.function {
-       TYPO3\CMS\Info\Controller\TranslationStatusController = 0
-    }
+    TCAdefaults.pages.hidden = 0
+    TCEMAIN.table.pages.disableHideAtCopy = 1
 
-If we activate this configuration in the TSconfig of a certain backend user, that
-user would still be able to select this menu item because the value of his user TSconfig
-overrides the same value set in the page TSconfig, just prefixed with
-:typoscript:`page.`:
+If we activate the following configuration in the user TSconfig of a certain backend
+user group, new and copied pages will be hidden for that group. The user TSconfig to
+be used is the same, but prefixed with :tsconfig:`page.`
 
 ..  code-block:: typoscript
-    :caption: EXT:site_package/Configuration/user.tsconfig
+    :caption: EXT:site_package/Configuration/TSconfig/editors.tsconfig
```

Comment:
Why change the caption? user.tsconfig is the preferred way in v13+, as this is added automatically and cached (without manual import).

For v12 backport you can change this back if you like, but then use "TsConfig" in path as outlined here:
file:///home/chris/Projects/TYPO3CMS-Reference-CoreApi/Documentation-GENERATED-temp/ExtensionArchitecture/FileStructure/Configuration/TsConfig/Index.html

Reply:
It is the preferred way to add **global** user TSconfig. However in this example only the TSconfig for a certain group should be set.

Reply:
Yup, do it already this way in v12 -> condition. So, wrap it around a condition and everything is fine.

Reply:
You can set the user TS config globally, but you can also still set it on group basis if that fits the use case. Setting user TS config directly in the group is neither deprecated nor discouraged. The only thing that got replaced was setting global / default user TSconfig.

Reply:
Okay, I don't care about it anymore.